DiceKeys/Views/Scanning/CameraController.swift

```diff
@@ -199,6 +204,12 @@ final class DiceKeysCameraController: NSObject, AVCaptureVideoDataOutputSampleBu
             layer.insertSublayer(previewLayer, at: 0)
         }
         previewLayer.frame = view.frame
+        #if os(macOS)
+        if let connection = previewLayer.connection, connection.isVideoMirroringSupported {
+            connection.automaticallyAdjustsVideoMirroring = false
+            connection.isVideoMirrored = true
```

Comment:
This turns on video mirroring for display but off for the frame grabber?  Again, a comment would be helpful!

Reply:
Sounds good.  #130